OpenTabletDriver.Desktop/Interop/Input/Absolute/WindowsAbsolutePointer.cs

```diff
@@ -10,11 +10,24 @@ namespace OpenTabletDriver.Desktop.Interop.Input.Absolute
 
     public class WindowsAbsolutePointer : WindowsVirtualMouse, IAbsolutePointer
     {
+        static WindowsAbsolutePointer()
+        {
+            timer = new System.Threading.Timer(state =>
+            {
+                ScreenToVirtualDesktop = new Vector2(SystemInterop.VirtualScreen.Width, SystemInterop.VirtualScreen.Height) / 65535;
+            }, null, TimeSpan.Zero, TimeSpan.FromSeconds(1));
```

Comment:
Would like to see this cleaned up, but not gonna be a stickler about it.
```suggestion
            timer = new System.Threading.Timer(
                state => ScreenToVirtualDesktop = new Vector2(
                    SystemInterop.VirtualScreen.Width,
                    SystemInterop.VirtualScreen.Height
                ) / 65535,
                null,
                TimeSpan.Zero,
                TimeSpan.FromSeconds(1)
            );
```